### DISCUSSION TITLE
Loading a bukkit world [solved]

### DISCUSSION THREAD

**Discussion by HttpPython** [Minestom/Minestom#1623] [Help]
well i am currently trying to load a bukkit world into minestom and i only get a void world

here is my code

    public void loadWorld(String world, String path) {
        Path finalPath = Path.of(path);
        AnvilLoader anvilLoader = new AnvilLoader(finalPath);
        InstanceContainer instanceContainer = MinecraftServer.getInstanceManager().createInstanceContainer();
        instanceContainer.setChunkLoader(anvilLoader);
        instances.put(world, instanceContainer);
    }

**Comment by Timofffee** [comment]
I don't know what the last line means, but I think it's not so important. the code does not raise any questions. the question is more, do you specify that the player should be in this instance?

**Reply by HttpPython** [reply]
yes but i only spawn in the void

**Comment by Distractic** [comment]
I've had this problem before.
My world was a 1.8 world. 
So I went to load it solo in 1.19 and then I used it as a server world and the problem was solved

**Reply by HttpPython** [reply]
okay i will try it out

**Reply by HttpPython** [reply]
it works thank you!
